김진/231224/실패율.js

```diff
@@ -0,0 +1,31 @@
+function solution(N, stages) {
+    const failList = [];
+    const sortList = [];
+    const result = [];
+    for (let i = 0; i < N+1; i++) { //스테이지 길이만큼 배열 생성
+      failList.push(0);
+    }
+    for (let j of stages) { //클리어하지 못한 플레이어 수 배열에 더해주기
+      failList[j-1] += 1;
+    }
+    for (let k = 0; k < N+1; k++) { 
+      user = failList[k];
+      failList[k] = (stages.length === 0) ? 0 : (failList[k] / stages.length);
+      sortList.push(failList[k]); //0 나누기 0 예외처리 후 실패율 반환한 값
+      for (let l = 0; l < user; l++) {
+        stages.pop();
+      }
+    }
+    failList.pop(); //전부 클리어한 사용자 수 제외
+    sortList.pop();
+    sortList.sort((a, b) => a - b).reverse(); //내림차순 정렬
```

Comment:
```suggestion
    sortList.sort((a, b) => b- a)
```
이렇게 하면 내림차순을 정렬을 더 간단하게 할 수 있습니다.